mithril-explorer/src/utils.js

```diff
@@ -78,6 +78,20 @@ function getCExplorerUrlForPool(network, partyId) {
   return url;
 }
 
+function calculatesProcessDuration(startTime) {
```

Comment:
Maybe name it differently:
```suggestion
function formatProcessDuration(startTime) {
```